jax/_src/nn/functions.py

```diff
@@ -1417,3 +1417,21 @@ def scaled_dot_general(
   )
 
   return out
+  @jax.jit
+  def prelu(x: ArrayLike, a: ArrayLike) -> Array:
+    """
+    Applies the PReLU activation function element-wise.
+
+    Args:
+        x (jnp.ndarray): Input tensor.
+        a (jnp.ndarray): Learnable parameter.
```

Comment:
Please use two-space indentations. Also "learnable parameter" is not particularly descriptive – maybe say "slope parameter" or something similar?